docs/_v8/2_clientconfig.md

```diff
@@ -4,12 +4,19 @@ identification: clientconfig
 layout: default
 ---
 
-ClientConfig is a container for all the connection specific paramters that you can set. All attributes, except _TechnicalSenderID_, have default values.  The technical sender id is a required parameter to contstruct the class.
+ClientConfig is a container for all the connection specific paramters that you can set.
 
 {% highlight csharp%}
 
- // The sender id can be retrieved from your Digipost organisation account (https://www.digipost.no/bedrift)
-const string senderId = "123456";
-var config = new ClientConfig(senderId, Environment.Production);
+// The actual sender of the message. The broker is the owner of the organization certificate 
+// used in the library. The broker id can be retrieved from your Digipost organization account.
+var broker = new Broker(12345);
+
+// The sender is what the receiver of the message sees as the sender of the message. 
+// If you are delivering on behalf of yourself, set this to your organization`s sender id.
```

Comment:
Kanskje få frem noe sånt som:
If you are sending on behalf of yourself senderId will be the same as brokerId.